build/NuSpecs/MUXControlsFrameworkPackage.nuspec

```diff
@@ -29,9 +29,13 @@
     <file target="tools" src="VisualStudioToolsManifest.xml"/>
 
     <file target="build" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\FrameworkPackage\MicrosoftUIXamlVersion.props"/>
+    <file target="buildTransitive" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\FrameworkPackage\MicrosoftUIXamlVersion.props"/>
     <file target="build\Common.targets" src="MUXControls-Nuget-Common.targets"/>
+    <file target="buildTransitive\Common.targets" src="MUXControls-Nuget-Common.targets"/>
     <file target="build\$ID$.props" src="MUXControls-Nuget-FrameworkPackage.props"/>
+    <file target="buildTransitive\$ID$.props" src="MUXControls-Nuget-FrameworkPackage.props"/>
     <file target="build\native\$ID$.targets" src="MUXControls-Nuget-FrameworkPackage.targets"/>
```

Comment:
Does buildTransitive not support native projects? Asking because I would have expected this line to be duplicated too...

Reply:
TBH I have no idea if this applies to nuget packages with native libs. Rather than guessing I left it out. Perhaps @karann-msft would know?